src/main/java/newbank/server/commands/RegisterCommand.java

```diff
@@ -40,7 +40,7 @@ public String execute() {
     } catch (DuplicateCustomerException ex) {
       return "FAIL: " + String.format("Customer name [%s] already exists.", username);
     } catch (PasswordInvalidException e) {
-      return "FAIL: Specified password does not meet the security requirements.";
+      return "FAIL: Password must contain mixture of uppercase and lowercase letters and at least one number.";
```

Comment:
Minor: must contain *a* mixture

Regards,
Jules

Reply:
thanks pushing update now

Reply:
Thanks @claudist, the tests now look green!